common/app/views/fragments/header.scala.html

```diff
@@ -19,9 +19,7 @@
             aria-label="Guardian sections">
 
             <a href="@LinkTo {/}"
-            class="@{if(AnniversaryLogoHeader.isSwitchedOn) {"new-header__logo news-header__logo-anniversary"} else {
```

Comment:
We're removing `news-header__logo-anniversary` here but the css is still extant in the file. I see the [original PR](https://github.com/guardian/frontend/pull/23748) that added this css had a specific purpose so we should be careful to respect that (this is more a note to self at this point!)